packages/react-redux-subspace/src/hooks/useSubspace.js

```diff
@@ -6,31 +6,31 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { useContext, useMemo } from "react"
+import { useRef, useMemo, useContext } from "react"
 import { ReactReduxContext } from "react-redux"
 import { subspace } from "redux-subspace"
-import isObject from '../utils/isObject'
+import isObject from "../utils/isObject"
+import isFunction from "../utils/isFunction"
 
-export const useSubspaceAdvanced = (
```

Comment:
Sorry, I know you don't want to, but this is a breaking change and this needs to be a major bump. semver can be inconvenient... but its there for a reason. The upgrade will be easy for most (all) dependents

Reply:
I guess it begs the question, can we leave leave this function here and deprecate it instead of removing it?

Reply:
This function was never exposed out if the index.js, so I'm not worried about removing it. The breaking change is the decorator prop that you could pass to `SubspaceProvider` that no longer exists that was the cause for this function to exist.

Reply:
Note that that prop was never documented and was essentially an internal performance optimisation when using the HOC interface

Reply:
I was sure i followed this out the index exports... guess its too early in the morning to be reviewing.

I'll leave it up to you, but for me, the public interface is changing... its a major bump.

Reply:
Unfortunately I feel the same way, but I feels like the negative kind of churn for such a small break.

Reply:
The upgrade path is negligible... I wouldn't worry too much about it. Anyone on v4/5 can just move to it easily, and anyone older than that probably has more work with the other dependencies to upgrade to meet out newer peer deps.

Reply:
That's probably true.